src/riak_cs_app.erl

```diff
@@ -90,7 +91,7 @@ fetch_and_cache_admin_creds(Key) ->
             {ok, Obj} ->
                 User = riak_cs_user:from_riakc_obj(Obj, false),
                 Secret = User?RCS_USER.key_secret,
-                lager:info("setting admin secret as ~s", [Secret]),
+                _ = lager:debug("admin secret: ~s", [Secret]),
```

Comment:
It's bad security practice to record keys in logs - this line should be commented out so a developer has to explicitly edit the file to get it logged ONLY if they're trying to debug this specific code